pulp_smash/tests/rpm/utils.py

```diff
@@ -17,25 +13,6 @@ def set_up_module():
     utils.skip_if_type_is_unsupported('rpm')
 
 
-def gen_erratum():
```

Comment:
All of this support code was used by `UploadErratumTestCase` and is no longer needed.